scripts/devxp-clone.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Define text color variables
+GREEN=$(tput setaf 2)
+YELLOW=$(tput setaf 3)
+NC=$(tput sgr0) # No Color
+
+# Read the user from the file
+USER=$(cat $HOME/.local/devxp/data/.user)
+
+echo "${GREEN}Cloning repository to $HOME/.local/devxp...${NC}"
+git clone https://github.com/elhub/devxp-linux.git $HOME/.local/devxp/devxp-linux
```

Comment:
This needs to be changed to `git clone -b move https://github.com/elhub/devxp-linux.git $HOME/.local/devxp/devxp-linux` if local testing is performed. Otherwise you will not pull this branch